.github/workflows/build-and-test.yml

```diff
@@ -1,17 +1,24 @@
-name: Build and Test
+name: Build and Test Nightly
 on:
+  workflow_dispatch:
   pull_request:
     branches: [master]
   push:
     branches: [master]
 jobs:
-  build-and-test:
+  build-and-test-nightly:
```

Comment:
Cither change the filename into `build-and-test-nightly.yml` or rename this into `build-and-test`.